include/mppic/optimizers/xtensor/critics/prefer_forward_critic.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 FastSense, Samsung Research
+#pragma once
+
+
+#include "mppic/optimizers/xtensor/critic_function.hpp"
+#include "mppic/utils.hpp"
+
+namespace mppi::xtensor::critics
+{
+
+class PreferForwardCritic : public CriticFunction
```

Comment:
Is it not possible to share the critics across the different optimizer plugins? If the bulk of the controller's code cannot be shared in that situation, I don't think that we should be including it.

Reply:
No it's not possible, i have the same feeling that it is a bad idea to have optimizer plugins